plugins/tinymcespellchecker.md

```diff
@@ -47,68 +49,189 @@ tinymce.init({
+
+**Type:** `String[]`
+
+#### Example: `spellchecker_whitelist`
+
+```js
+tinymce.init({
+  selector: 'textarea',
+  plugins: 'tinymcespellchecker',
+  spellchecker_whitelist: ['tinymce','TinyMCE']
+});
+```
 
 ### `spellchecker_on_load`
-This option lets run the spellchecker when the contents is loaded into the editor. This option defaults to "false" so it's disabled by default.
+
+> **Note**: Removed in Spell Checker Pro 2.0 (TinyMCE 5.2). Spell Checker Pro will now always run on editor initialization. To disable Spell Checker Pro on load, use [`spellchecker_active`](#spellchecker_active).
+
+This option runs the spellchecker when the contents of the editor is loaded.
+
+**Type:** `Boolean`
+
+**Default Value:** `false`
+
+**Possible Values:** `true`, `false`
+
+#### Example: `spellchecker_on_load`
+
+```js
+tinymce.init({
+  selector: 'textarea',
+  plugins: 'tinymcespellchecker',
+  spellchecker_on_load: true
+});
+```
 
 ### `spellchecker_active`
-This option helps in deciding if the spellchecker should be initialized as active or not. With this set to `false` the spellchecker will not be activated automatically on text input. The toolbar button or the menu item will have to be pressed to start the spellchecking. Only applicable when using the default as-you-type spellchecking, not while using the dialog mode. Defaults to `true`.
+
+This option enables or disables the spell checker when the editor is loaded. When set to `false`, the spellchecker will not be active when the editor is initialized. The toolbar button or the menu item will have to be selected by the user to start the spell checker.
+
+**Type:** `Boolean`
+
+**Default Value:** `true`
+
+**Possible Values:** `true`, `false`
+
+#### Example: `spellchecker_active`
+
+```js
+tinymce.init({
+  selector: 'textarea',
+  plugins: 'tinymcespellchecker',
+  spellchecker_active: true
+});
+```
 
 ## Toolbar Buttons
 
-### `spellchecker`
+The following toolbar buttons are provided by the Spell Checker Pro plugin:
 
-This button allows the user to perform a spellcheck on the entire document. In addition, the drop-down menu attached to this button allows specifying the language that is currently used when spellchecking. For more information about customizing the toolbar, refer to the [Editor appearance section of the documentation]({{site.baseurl}}/configure/editor-appearance/#toolbar).
+| Control              | Description                                                                           |
+| -------------------- | ------------------------------------------------------------------------------------- |
+| language             | Sets the spelling checker language for the current selection.                         |
+| spellcheckdialog     | Opens the spelling checker dialog.                                                    |
+| spellchecker         | Spellchecks the current editor contents.                                              |
```

Comment:
Should these account for the toolbar button behaviour changes from spellchecker_dialog?

Reply:
fixed?